homeassistant/components/nissan_leaf/sensor.py

```diff
@@ -18,9 +18,6 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Sensors setup."""
-    _LOGGER.debug("setup_platform nissan_leaf sensors, discovery_info=%s",
-                  discovery_info)
-
     devices = []
     for key, value in hass.data[DATA_LEAF].items():
```

Comment:
Clarify variable names as above.

Reply:
Fixed in commit https://github.com/home-assistant/home-assistant/pull/21145/commits/daca6ede4711c75f55d971aed1acdf3311c4553c